packages/next/src/server/lib/router-utils/setup-dev-bundler.ts

```diff
@@ -290,27 +293,40 @@ async function startWatcher(opts: SetupOpts) {
     }
 
     function formatIssue(issue: Issue) {
-      const { filePath, title, description, source, detail } = issue
-      let formattedTitle = title.replace(/\n/g, '\n    ')
+      const { filePath, title, description, source } = issue
+      let { documentationLink } = issue
 
       let formattedFilePath = filePath
-        .replace('[project]/', '')
+        .replace('[project]/', './')
         .replaceAll('/./', '/')
         .replace('\\\\?\\', '')
 
-      let message
+      let message, ignoreColumns
 
       if (source) {
         if (source.range) {
           const { start } = source.range
-          message = `${issue.severity} - ${formattedFilePath}:${
-            start.line + 1
-          }:${start.column}`
+          message = `${formattedFilePath}:${start.line + 1}:${start.column}`
         } else {
-          message = `${issue.severity} - ${formattedFilePath}  ${formattedTitle}`
+          message = formattedFilePath
         }
       } else {
-        message = `${formattedTitle}`
+        message = title.replace(/\n/g, '\n    ')
```

Comment:
It's a bit weird that we only show the `title` or the error when there is not `source`. I guess the title should always be added to the message.